### PR TITLE
Adding additional keyboard shortcuts for the Windows Company Portal

### PR DESCRIPTION
**Pull request by allydurks**
While looking at the docs for the keyboard shortcuts for the Windows Company Portal, we noticed there were some missing. Specifically: the app tile and app list items have the shortcut Ctrl+I for install.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@allydurks : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by allydurks** [comment]
Linking #1582

**Comment by PRMerger10** [comment]
@allydurks : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/apps/company-portal-app.md [hunk 1]
```diff
@@ -228,6 +228,8 @@ The following keyboard shortcuts are available in the Windows Company Portal app
 |  | Remove | Ctrl+D or Delete |
 |  | Check access | Ctrl+M or F9 |
 | App details | Install | Ctrl+I |
+| Apps list tile | Install | Ctrl+I |
+| Apps list item | Install | Ctrl+I |
 | Devices | Available | Ctrl+D |
```
Comment (kagillig) [inline memdocs/intune/apps/company-portal-app.md:233]:
```suggestion
```

**Comment by PRMerger18** [comment]
@allydurks : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@allydurks : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Pull request merged**